src/commands/env/list.js

```diff
@@ -39,9 +39,9 @@ class EnvListCommand extends Command {
     const table = new AsciiTable(`Environment variables`)
 
     table.setHeading('Key', 'Value')
-    for (const [key, value] of Object.entries(environment)) {
+    Object.entries(environment).forEach(([key, value]) => {
```

Comment:
Same as other comment about `addRowMatrix`

Reply:
Okay. `addRowMatrix` seems better for both instances of bulk row operations.